src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Container, CssBaseline, Stack, Typography,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import DefaultHeader from '../../common/DefaultHeader/DefaultHeader';
+import { CustomBox, MainLockIcon, Wrapper } from './Auth.style';
+import AuthSwitchElement from './AuthSwitchElement/AuthSwitchElement';
+import Login from './Login/Login';
+import Registration from './Registration/Registration';
+
+const Auth = () => {

```

Comment:
Why we need a new component if we already have Page for it?

Reply:
fixed